tests/e2e/e2e_k8s_version_test.go

```diff
@@ -24,8 +24,8 @@ var _ = OSMDescribe("Test HTTP traffic for different k8s versions",
 		Context("Version v1.18.15", func() {
 			testK8sVersion("v1.18.15")
 		})
-		Context("Version v1.17.17", func() {
-			testK8sVersion("v1.17.17")
+		Context("Version v1.18.0", func() {
```

Comment:
We can remove v1.18.15 above, we just wanted to run against 1 minor version here. We anyway aren't running this test against every patch version.